Cargo.toml

```diff
@@ -31,6 +31,7 @@ members = [
     "test-utils",
     "parquet-test-utils",
     "benchmarks",
+    "tests/sqllogictests",
```

Comment:
I recommend moving this test into `datafusion/core/tests` so that it would then be run via

```shell
cargo test -p datafusion --test sqllogictests
```

I don't see any reason to put it into its own top level crate (though if others feel differently perhaps we could move the code into `datafusion/sqllogictest` to match the structure of the other crates in this repo.

Reply:
Well TIL about cargo tests' `harness = false`! Thanks for the tip